src/middlewared/middlewared/plugins/update.py

```diff
@@ -268,6 +268,23 @@ def get_trains(self):
             'selected': selected,
         }
 
+    @accepts(Str('train'))
+    def set_train(self, train):
+        """
```

Comment:
Don't we want any check here? Like in `update`

Reply:
We probably do, good thought

Reply:
What is a point of `call("update.set_train", null)`?